src/app/server/CommissioningWindowManager.cpp

```diff
@@ -155,8 +155,10 @@ void CommissioningWindowManager::OnSessionEstablished()
 
     DeviceLayer::PlatformMgr().AddEventHandler(OnPlatformEventWrapper, reinterpret_cast<intptr_t>(this));
 
-    StopAdvertisement(/* aShuttingDown = */ false);
     ChipLogProgress(AppServer, "Device completed Rendezvous process");
+
+    // Restart the PASE session in case something happens to the commissioner and it needs to re-establish
+    AdvertiseAndListenForPASE();
```

Comment:
The place to stop advertising as commissionable is after the commissining complete command is received. Because during the commissioning process, the device is still commissionable.